src/matter/cluster/GroupKeyManagementCluster.ts

```diff
@@ -0,0 +1,135 @@
+/**
+ * @license
+ * Copyright 2022 The node-matter Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { WritableAttribute, Attribute, Cluster, Command, TlvNoArguments, TlvNoResponse } from "./Cluster";
+import { StatusCode } from "../interaction/InteractionMessages";
+import { TlvGroupId } from "../common/GroupId";
+import { BitFlag, MatterApplicationClusterSpecificationV1_0, TlvArray, TlvEnum, TlvField, TlvNullable, TlvObject, TlvString, TlvUInt16, TlvInt64 } from "@project-chip/matter.js";
+import { TlvEndpointNumber } from "../common/EndpointNumber";
+
+
+/** @see {@link MatterApplicationClusterSpecificationV1_0} § 11.2.6.1 */
+const TlvGroupKeyMapStruct = TlvObject({ 
+    groupId: TlvField(0, TlvGroupId), /* min: 1 */
+    groupKeySetId: TlvField(1, TlvUInt16),
+});
+
+/** @see {@link MatterApplicationClusterSpecificationV1_0} § 11.2.6.2 */
+const TlvGroupKeySetStruct = TlvObject({
+    groupKeySetID: TlvField(0, TlvUInt16),
+    groupKeySecurityPolicy: TlvField(1, TlvUInt16),
+
+    epochKey0: TlvField(2, TlvNullable(TlvString.bound({ maxLength: 16 }))),
+    epochStartTime0: TlvField(3, TlvNullable(TlvInt64)), // epoch_us
+
+    epochKey1: TlvField(4, TlvNullable(TlvString.bound({ maxLength: 16 }))),
+    epochStartTime1: TlvField(5, TlvNullable(TlvInt64)), // epoch_us
+ 
+    epochKey2: TlvField(6, TlvNullable(TlvString.bound({ maxLength: 16 }))),
+    epochStartTime2: TlvField(7, TlvNullable(TlvInt64)), // epoch_us
+});
```

Comment:
Specs know a provisional field GroupKey­Multicast­Policy ... lets add as optional field ... (plus the new enum)